libpod/runtime.go

```diff
@@ -131,6 +131,8 @@ type RuntimeConfig struct {
 	CNIPluginDir []string `toml:"cni_plugin_dir"`
 	// HooksDir Path to the directory containing hooks configuration files
 	HooksDir string `toml:"hooks_dir"`
+	// DefaultMountsFile is the path to the default mounts file for testing purposes only
+	DefaultMountsFile string `toml:"default_mounts_file"`
```

Comment:
We probably don't want this in the generated TOML configs, so `toml:"-"` is better

Reply:
Also, can we make this a []string here, so we don't have to convert every time we mount a container? WithDefaultMountsFile() can still only accept a single argument, it'll just package it into a []string when it puts it here

Reply:
okay, Fixed.